packages/platform-ios/src/link-pods/registerNativeModule.js

```diff
@@ -31,5 +35,21 @@ function getLinesToAddEntry(podLines, {projectName}) {
     return linesToAddPodWithMarker;
   }
   const firstTargetLined = findPodTargetLine(podLines, projectName);
+  if (firstTargetLined === null) {
+    throw new CLIError(
+      inlineString(`
+        We couldn't find a target to add a CocoaPods dependency.
+        
+        Make sure that you have a ${chalk.dim(
```

Comment:
can we add `"` quotes around the dimmed lines so they're easier distinguishable for users with poor colors?

Reply:
Done! Good idea.